src/Helpers/Constants.cs

```diff
@@ -0,0 +1,12 @@
+using System.Globalization;
+
+public class Constants
+{
+    public static readonly long MINIMUM_CHANNEL_CAPACITY_SATS;
+    
+    static Constants()
+    {
+        var environmentVariable = Environment.GetEnvironmentVariable("MINIMUM_CHANNEL_CAPACITY_SATS") ?? throw new InvalidOperationException("MINIMUM_CHANNEL_CAPACITY_SATS not set");
```

Comment:
I like this, we can migrate step-by-step to this new approach

Reply:
I'll create an issue